tmt-app/src/Components/Stepper.jsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { useLocation } from "react-router-dom";
+import { CustomTheme } from "../Styling/CustomStyling";
+import HomeIcon from "../Images/icons/boba_progress/Home_Current.png";
+import SubjectsIcon from "../Images/icons/boba_progress/Subjects_Current.png";
+import SoftSkillsIcon from "../Images/icons/boba_progress/SoftSkills_Current.png";
+import OrderUpIcon from "../Images/icons/boba_progress/OrderUp_Current.png";
+import {
+  Stepper,
+  Step,
+  StepLabel,
+  StepConnector,
+  stepConnectorClasses,
+  styled,
+} from "@mui/material";
+
+const ColorlibConnector = styled(StepConnector)(({ theme }) => ({
+  [`&.${stepConnectorClasses.active}`]: {
+    [`& .${stepConnectorClasses.line}`]: {
+      backgroundColor: "#1B1D2E",
+    },
+  },
+  [`&.${stepConnectorClasses.completed}`]: {
+    [`& .${stepConnectorClasses.line}`]: {
+      backgroundColor: "#1B1D2E",
+    },
+  },
+  [`& .${stepConnectorClasses.line}`]: {
+    height: 12,
+    border: 0,
+    backgroundColor: CustomTheme.palette.BobaHighlight.main,
+    borderRadius: 0,
+  },
+}));
+
+const ColorlibStepIconRoot = styled("div")(() => ({
+  zIndex: 1,
+  height: 40,
+  display: "flex",
+  justifyItems: "center",
+  alignItems: "center",
+  overflow: "visible",
+}));
```

Comment:
do we need a styled component here? we could probably just add in these values as sx props. Is there value you see by adding this in like this?